sdk/eventhub/azure-eventhub/azure/eventhub/_common.py

```diff
@@ -101,6 +107,14 @@ def __init__(self, body=None):
         self.message.annotations = {}
         self.message.application_properties = {}
 
+        # Internal usage only for transforming AmqpAnnotatedMessage to outgoing ServiceBusMessage
+        self._raw_amqp_message = AmqpAnnotatedMessage(message=self.message)
+        self.message = self._raw_amqp_message._message  # pylint:disable=protected-access
+        self._raw_amqp_message.header = AmqpMessageHeader()
+        self._raw_amqp_message.properties = AmqpMessageProperties()
```

Comment:
we could rewrite the `__init__` to replace uamqp.message construction with directly the AmqpAnnotatedMessage construction -- AmqpAnnotatedMessage will read the message and construct again which wastes cycles.